SoftLayer/managers/block.py

```diff
@@ -201,6 +201,40 @@ def deauthorize_host_to_volume(self, volume_id,
         return self.client.call('Network_Storage', 'removeAccessFromHostList',
                                 host_templates, id=volume_id, **kwargs)
 
+    def assign_subnets_to_acl(self, access_id, subnets_id):
+        """Assigns subnet records to ACL for the access host.
+
+        :param integer access_id: id of the access host
+        :param integer subnets_id: The ids of the subnets to be assigned
```

Comment:
Shouldn't this be a list? Might help to have some code here to check/cast `subnets_id` to a list unless sending in a single value works as well. 
I'd recommend using `subnet_ids` as that reads a bit better.

Reply:
1- Correct, these features are only in Block. Files does not have them.

2- Thanks, will update.

3- i provided how to get access_id on the docblock:
"access_id is the allowed_host_id from slcli block access-list"
I can maybe add a -h with details.

4- Thanks, will update.

Reply:
i am casting subnets_id  to list on the softlayer/cli/block/subnets/assign|remove.py files.
so there shouldnt be a need to cast again on manager/block.py